src/oca_github_bot/tasks/merge_bot.py

```diff
@@ -96,18 +132,7 @@ def merge_bot_status(org, repo, merge_bot_branch, sha):
         with github.login() as gh:
             gh_repo = gh.repository(org, repo)
             gh_commit = github.gh_call(gh_repo.commit, sha)
-            gh_status = github.gh_call(gh_commit.status)
-            # here we assume the status is complete!
-            success = None
-            for status in gh_status.statuses:
-                if status.context in ("ci/runbot", "codecov/project", "codecov/patch"):
-                    # ignore
-                    continue
-                if status.state == "success":
-                    success = True
-                else:
-                    success = False
-                    break
+            success = _get_commit_success(gh_commit)
             if success is True:
```

Comment:
Isn't enough?
```suggestion
            if success:
```

Reply:
no, because success is None means it's still in progress

Reply:
Yeah, I know, but for truly expression (`None` is evaled as falsy), it doesn't matter.